src/main/java/com/minecolonies/coremod/client/gui/townhall/WindowInfoPage.java

```diff
@@ -86,14 +87,14 @@ private void createAndSetStatistics()
         final int citizensSize = building.getColony().getCitizens().size();
         final int citizensCap;
 
-        if(MinecoloniesAPIProxy.getInstance().getGlobalResearchTree().hasResearchEffect(CITIZEN_CAP))
+        if (MinecoloniesAPIProxy.getInstance().getGlobalResearchTree().hasResearchEffect(CITIZEN_CAP))
         {
-            citizensCap = (int) (Math.min(MineColonies.getConfig().getServer().maxCitizenPerColony.get(),
-              25 + this.building.getColony().getResearchManager().getResearchEffects().getEffectStrength(CITIZEN_CAP)));
```

Comment:
is it intended to drop the static +25 here?

Reply:
I believe that is intended, given the way that the `CITIZEN_LIMIT_*` constants are defined.  However as is it's not quite correct, as per my comment below.

Reply:
It'll always be limited by the `CITIZEN_LIMIT_DEFAULT`.

```
final int max = Math.max(CitizenConstants.CITIZEN_LIMIT_DEFAULT, (int) this.building.getColony().getResearchManager().getResearchEffects().getEffectStrength(CITIZEN_CAP));
```
Ensures that the highest between the research, or the default is taken, it the research is not yet done, it'll return 0, meaning the max allowed citizen count will be 25, afterwards the config is taken to determine the minimum.